src/styles/styles.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "styled-components";
+import "./styles.css";
+
+// Colors
+export const YELLOW = "#ECF955";
+export const GREEN = "#058855";
+export const PINK = "#E8A8CA";
+export const RED = "#DF4E20";
+export const ROYAL_BLUE = "#3C60DD";
+export const LIGHT_BLUE = "#13BCD7";
+export const BLACK = "#170E07";
+export const CREAM = "#FFFAF2";
+
+export const fonts = {
+  cubano: "Cubano",
+  epilogue: "Epilogue",
+  epilogueItalic: "Epilogue Italic",
```

Comment:
don't think we need this anymore!! that's my bad I had 2 separate `font-face`s for epilogue in the first fonts PR but I changed it the second time around